File: searchingAlgorithms/linearSearch/LinearSearch.py
def search(arr, ans):
	i = 0
	while i < len(arr):
		if(arr[i] == ans):
			return i

		i = i + 1
		
	return -1
	

arr = [1,2,3,4,5]

index = search(arr, 8)

if index == -1:
	print("element not found")
else:
	print("element found at index ")
	print(index)

File: searchingAlgorithms/linearSearch/LinearSearch.java
import java.util.Scanner;

public class LinearSearch{

	public static int Search(int[] arr, int find){

		for(int i = 0; i < arr.length; i++){

			if(arr[i] == find){

				return i;

			}

		}

		return -1;

	}

	public static void main(String[] args){

		int[] arr = {6, 7, 1, 13, 6, 18, 20};

		Scanner reader = new Scanner(System.in);

		System.out.println("What number do you want to search for in the array");

		int find = reader.nextInt();
		reader.close();

		int ans = Search(arr, find);

		if(ans == -1){

			System.out.println("Integer was not found in array");

		}else{

			System.out.println("Integer was found at index " + ans);

		}

	}

}